api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -1580,6 +1580,68 @@ paths:
         - OAuth2:
             - SIG/ALL
 
+  /signals/v1/private/reports/signals/open/:
+    get:
+      description: Returns a total count of Signals in a "open" state + a count per category
+      parameters:
+        - name: "start"
+          in: query
+          description: Filters Signals created after this datetime
+          schema:
+            type: string
+          required: false
+        - name: "end"
+          in: query
+          description: Filters Signals created before this datetime
+          schema:
+            type: string
+          required: false
+      responses:
+        '200':
+          description: Returns a total count of Signals in a "open" state + a count per category
```

Comment:
Nit: Consider adding something like "open" state _and past more than three times the allotted time to resolution_

Point being it is not just open signals, it is open and too late.